src/Helper/OrganisationApiHelper.php

```diff
@@ -40,8 +40,9 @@ public function getBrugerInformationer(string $brugerId): array {
    * @throws \Drupal\os2forms_organisation\Exception\ApiException
    *   API exception.
    */
-  public function getFunktionInformationer(string $brugerId): array {
-    $response = $this->get('bruger/' . $brugerId . '/funktioner');
+  public function getFunktionInformationer(string $brugerId, bool $findManagerIds = FALSE): array {
+    $path = $findManagerIds ? 'bruger/' . $brugerId . '/leder-funktioner' : 'bruger/' . $brugerId . '/funktioner';
```

Comment:
Consider rewriting to

```suggestion
    $path = 'bruger/' . $brugerId . ($findManagerIds ? '/leder-funktioner' : '/funktioner');
```